bears/general/KeywordBear.py

```diff
@@ -113,3 +117,25 @@ def run(self,
                     end_column=keyword.end() + 1,
                     severity=RESULT_SEVERITY.INFO,
                     diffs=diffs)
+
+            if regex_keyword is not '':
```

Comment:
I'd make two for loops actually so we can move the if outside the loop:

```python
keywords_regex = ...
if keywords:
    for ...:
        ...

if regex_keyword:
    for ...:
        ...
```

Note that I also added the `if keywords` checking the normal keywords for emptiness!

And if we have that we see, that we have a huge chunk of duplication here, so you can actually move the whole if and for loop outside into a function that takes a regex and a custom message with one format argument, saving like half of the code here.